inlong-sort-standalone/sort-standalone-common/src/main/java/org/apache/inlong/sort/standalone/config/holder/CommonPropertiesHolder.java

```diff
@@ -196,22 +200,22 @@ public static Integer getInteger(String key) {
     public static String getClusterId() {
         return getString(KEY_CLUSTER_ID);
     }
-
+    
     /**
-     * Get manager URL
+     * getAuditFormatInterval
      *
-     * @return Manager URL
+     * @return
      */
-    public static String getSourceConfigManagerUrl() {
-        return getString(KEY_SOURCE_CONFIG_MANAGER_URL);
+    public static long getAuditFormatInterval() {
+        return auditFormatInterval;
     }
 
     /**
-     * getAuditFormatInterval
-     * 
+     * get if managerAddrNeedUpdate
+     *
      * @return
      */
-    public static long getAuditFormatInterval() {
-        return auditFormatInterval;
+    public static boolean isManagerAddrNeedUpdate() {
```

Comment:
The method "isManagerAddrNeedUpdate" is unusable.

Reply:
maybe this managerAddrNeedUpdate is useless, need further disscuss